### ISSUE TITLE
doc: release history webpage contains suboptimal links

### ISSUE THREAD

**Issue by AlekSi** [golang/go#35988]
### What did you do?

Opened https://golang.org/doc/devel/release.html#go1.13.minor containing this text:
> go1.13.5 (released 2019/12/04) includes fixes to the go command, the runtime, the linker, and the net/http package. See the Go 1.13.5 milestone on our issue tracker for details.

Followed a link to the GitHub milestone: https://github.com/golang/go/issues?q=milestone%3AGo1.13.5

### What did you expect to see?

Issues fixed in 1.13.5.

### What did you see instead?

Issues considered for 1.13.5, including the ones that were not backported. For example: https://github.com/golang/go/issues/34642

I think either @gopherbot should remove milestone from issues without `CherryPickApproved` label after release is done, or link in documentation should include this label.

See also https://github.com/golang/go/issues/35119#issuecomment-548728868 and below.

**Comment by dmitshur** [comment]
I agree the current link leads to a page that is hard to read. We shouldn't remove issues from that milestone, because then it will be harder to track history and know what was considered but not accepted. I think a better fix is to add the `CherryPickApproved` label filter to the link.

We've done this for some past releases (e.g., Go 1.12.9, 1.11.14, and 1.10.7) but not all. We need to adjust the release process so this happens for future releases.

In the mean time, please use this link:

https://github.com/golang/go/issues?q=milestone%3AGo1.13.5+label%3ACherryPickApproved

/cc @toothrot @cagedmantis @andybons

**Comment by dmitshur** [comment]
I went through the release history a bit. We started consistently using the CherryPickCandidate and CherryPickApproved labels with Go 1.8.1 and later. They weren't used before Go 1.8. I fixed up some minor inconsistencies (i.e., where a change was acutally cherry-picked but its backport issue was missing CherryPickApproved label) in 1.8-1.10; going through 1.11-1.13 releases is still a TODO.

Sent [CL 209918](https://golang.org/cl/209918) to add the `CherryPickApproved` label filter to links for all releases 1.8 for review.
